git_pile/git_pile.py

```diff
@@ -150,6 +164,20 @@ def revert(self, other):
     def destroy(self):
         git("config --remove-section pile", check=False, stderr=nul_f, stdout=nul_f)
 
+    @classmethod
+    def help(cls, prefix=""):
+        attr_prefix = "_Config__attr_doc_"
+        attr_docs = (k for k in cls.__dict__.keys() if k.startswith(attr_prefix + prefix))
+        if not attr_docs:
```

Comment:
`attr_docs` will never evaluate to false because it is a generator object, even if it would generate no entries. Example:

```python
>>> gen = (i for i in range(0))
>>> bool(gen)
True
>>> list(gen)
[]
```

Reply:
thanks